app/assets/javascripts/modules/track-email-alert-signup-choices.js

```diff
@@ -0,0 +1,43 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (global, GOVUK) {
+  'use strict'
+
+  var $ = global.jQuery
+
+  GOVUK.Modules.TrackEmailAlertSignupChoices = function () {
+    this.start = function (element) {
+      track(element)
+    }
+
+    function track (element) {
+      element.on('submit', function (event) {
+        var $checkedOption, eventLabel, options
+        var $submittedForm = $(event.target)
+        var $checkedOptions = $submittedForm.find('input:checked')
+        var category = $submittedForm.data('track-category')
+        var action = $submittedForm.data('track-action')
+
+        if ($checkedOptions.length) {
+          $checkedOptions.each(function (index) {
+            $checkedOption = $(this)
+            var checkedOptionId = $checkedOption.attr('id')
+            var checkedOptionLabel = $submittedForm.find('label[for="' + checkedOptionId + '"]').text().trim()
+            eventLabel = checkedOptionLabel.length
+              ? checkedOptionLabel
+              : $checkedOption.val()
+
+            options = { transport: 'beacon', label: eventLabel }
+
+            GOVUK.analytics.trackEvent(category, action, options)
```

Comment:
Now we're not using the wrapper, I think we need to check that GOVUK.analytics is available before we call it. (same comment for a couple of lines below too)